bin/__tests__/build-sprite.test.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-env jest */
+import buildSpriteString from '../build-sprite-string';
```

Comment:
Super minor, but this test should probably be called `build-sprite-string.test.js` because that's actually the function we're testing here.